Catty-166: SetLookBrick in "My first project"

neel-makhecha opened the pull request:
- Replaced "Switch to Look" brick to "Set Background" brick in the default project.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [ ] Include the name of the Jira ticket in the PR’s title
- [ ] Verify that the Jira ticket is in the status *Ready for Development*
- [ ] Include a summary of the changes plus the relevant context
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s git workflow (rebase and squash your commits)
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

m-herold commented:
Thanks a lot! Could you please shortly explain how you made those changes in the _.catrobat_ file?

neel-makhecha commented:
I had opened the project in the iPhone simulator and edited the script for background right from there. Then found the exact location in Finder, zipped it again, changed extension to .catrobat and replaced it with the existing one. (Before doing all this, I had done the exact reverse of this with the .catrobat file from Resources folder, by unzipping and observing it). 
After all this, it works as intended.

neel-makhecha commented:
Hi Michael,
I have updated the file. Please review it.
